WordPressAuthenticator/Signin/LoginViewController.swift

```diff
@@ -299,11 +299,15 @@ extension LoginViewController {
             return
         }
 
+        // FIXME: Tests this
```

Comment:
Woops. I shall followup on this ASAP...

Reply:
Done in [`ccdc610` (#778)](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/778/commits/ccdc6109870dd23c8e7ba9586fdd917c61c96831)